chore: drop Node 12 support

jrandolf-google opened the pull request:
This patches removes support for Node 12.

BREAKING CHANGE: Node 12 is no longer supported.

OrKoN commented:
Should we update package.json? https://github.com/puppeteer/puppeteer/blob/main/package.json#L14 Also, let's mark this PR as a breaking change?

jrandolf committed:
chore: drop Node 12 support

This patches removes support for Node 12.

BREAKING CHANGE: Node 12 is no longer supported.
